test/core/crypto/vrf/vrf_provider_test.cpp

```diff
@@ -43,12 +43,43 @@ TEST_F(VRFProviderTest, SignAndVerifySuccess) {
   // given
   VRFThreshold threshold{std::numeric_limits<VRFThreshold>::max() - 1};
   // when
-  auto out_opt = vrf_provider_->sign(msg_, keypair1_, threshold);
+  Buffer src{
```

Comment:
Might it be better to initialize it from a hex string to save space?

Reply:
Explicit buffer initialization from byte array much faster. More over it is much more simple to just copy this data from GDB printout :)

Reply:
My concern was that  it would hardly be that much faster to be ever noticed by a human being and you would need to put this data just once, while reading these tests is a much more often need and thus it might be more optimal to save the time for reading.

Reply:
Ok, I'll refactor that place.